_data/locales/en.yml

```diff
@@ -0,0 +1,276 @@
+--- 
+en: 
+  about:  "about"
+  about-maj:  "About"
+  all-posts:  "All blog posts"
+  around: "Approx."
+  back:  "Back"
+  by:  "by"
+  cancel:  "Reset"
+  category:
+    dad: "Dad"
+    web: "Web"
+    citizen: "Citizen"
+  citizen:  "citizen"
+  citizen-maj:  "Citizen"
+  closed-comments:  "Comments are closed."
+  comment:  "comment"
+  comment-link:  "Permalink to this comment"
+  comment-maj:  "Comment"
+  comment-post-success:  "You comment has been successfuly posted"
+  comments:  "comments"
+  comments-maj:  "Comments"
+  discuss:  "Do you want to talk about this post"
+  email-maj:  "Email"
+  english:  "english"
+  error-github:  "If you're a Githuber and you've detected a mistake"
+  follow-me:  "Follow me"
+  french:  "french"
+  from-category:  "from the category:"
+  goto-main:  "Go to main content directly"
+  home-maj:  "Home"
+  houston:  "Houston, we have a problem."
+  in-same-category:  "In the same category:"
+  issue:  issue
+  last-page:  "Last page"
+  leave-comment:  "Leave a comment"
+  modifation-of:  "modification of the"
+  name-maj:  "Name"
+  next-page:  "Next page"
+  next:  "Next:"
+  on:  "on"
+  only-web-related:  "If you only want to receive my web-relative posts, you might prefer to follow"
+  optional:  "optional"
+  or:  "or"
+  or-open:  or open an
+  papa:  "dad"
+  papa-maj:  "Dad"
+  planet:  "Coordinates of your planet (only for non-humans)"
+  please-share:  "If you enjoyed reading this, please share it"
+  please-submit:  "please submit a"
+  post-the-comment:  "Post the comment"
+  posts-in:  "Posts published in"
+  previous-page:  "Previous page"
+  previous:  "Previous:"
+  read-in:  "Read in"
+  read-more:  Read more
+  reading-time:  "min of reading time"
+  related-posts:  "You may also be interested by this posts"
+  reply-to:  "Reply to"
+  search:  "search"
+  search-maj:  "Search"
+  send-me-a:  "send me a"
+  sens-message:  "Join the discussion"
+  serie:  "Serie"
+  share-post:  "Share this post on"
+  site:  "Website"
+  source-code:  source code
+  subscribe-feed:  "Subscribe to the RSS feed"
+  the-category:  "the category"
+  this-feed:  "this feed"
+  web:  "web"
+  web-maj:  "Web"
+  write-me:  "Send me an email !"
+  written-in:  "written in"
+  you-re-subscribed:  "You subscribed to my posts"
+  date: 
```

Comment:
`jekyll-locale-0.4.0` provides a `localize_date` Liquid::Filter which expects `date` and `time` keys to be nested under `locale_date` key:
```yaml
locale_date:
  date:
      ...
  time:
      ...
```